plugin/storage/grpc/shared/mapping.go

```diff
@@ -0,0 +1,422 @@
+// Copyright (c) 2018 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package shared
+
+import (
+	"time"
+
+	"github.com/jaegertracing/jaeger/model"
+	"github.com/jaegertracing/jaeger/plugin/storage/grpc/proto"
+)
+
+func DependencyLinkSliceFromProto(p []*proto.DependencyLink) []model.DependencyLink {
```

Comment:
I would like to avoid the need for this file. The whole point of generating domain model from proto IDL was to be able to reuse those types in the grpc plugin interfaces